### DISCUSSION TITLE
bevy_imgui reserved crate name

### DISCUSSION THREAD

**Discussion by jbrd** [bevyengine/bevy#9639] [General]
Hello,

Just reaching out because I have recently written an ImGui integration for Bevy, and have made the repository available here: https://github.com/jbrd/bevy_imgui.

The next step for the above repository is to make it more generally available for people to try, and to facilitate this, it would be useful to be able to publish the crate. Ideally, in doing so, this repository would assume the bevy_imgui crate name, which is currently reserved. Could someone advise on the process required to take ownership of one of these reserved crate names?

Thanks!

**Comment by rlidwka** [comment]
> Could someone advise on the process required to take ownership of one of these reserved crate names?

From a practical standpoint, I would advise you to publish it as `bevy_mod_imgui`.

*source: similar experience with bevy_physx crate*

**Comment by konsti219** [comment]
There was some actual discussion about this on the discord in response to your initial issue opened on the crate reservations repo. https://discord.com/channels/691052431525675048/743663673393938453/1136020856863207485

**Comment by jbrd** [comment]
Thanks all - so, looking at the discussion that happened on the Discord channel, and the suggestion from @rlidwka above, I'd be happy to go with `bevy_mod_imgui` here - it feels like that would have a good level of consistency with other bevy plug-ins that provide external library integration
